spec/spec_helper.rb

```diff
@@ -17,12 +17,9 @@
 end
 
 require "lutaml/model"
-require "lutaml/model/xml_adapter/nokogiri_adapter"
-require "lutaml/model/json_adapter/standard_json_adapter"
-require "lutaml/model/yaml_adapter/standard_yaml_adapter"
 
 Lutaml::Model::Config.configure do |config|
-  config.xml_adapter = Lutaml::Model::XmlAdapter::NokogiriAdapter
-  config.json_adapter = Lutaml::Model::JsonAdapter::StandardJsonAdapter
-  config.yaml_adapter = Lutaml::Model::YamlAdapter::StandardYamlAdapter
+  config.xml_adapter_type = :nokogiri
```

Comment:
@HassanAkbar how about we just set defaults and the gem developer doesn’t need to set them?

Reply:
@ronaldtse We have default adapters set for `json` and `yaml`, so we can safely remove them. However, for `XML`, there is no default adapter set. There is an [open ticket](https://github.com/lutaml/lutaml-model/issues/309) in **lutaml-model** to determine the XML adapter based on the installed gems, but that hasn't been implemented yet.  

I'll update all the related PRs to remove the default adapters.